compiler-rt/lib/sanitizer_common/sanitizer_libignore.h

```diff
@@ -54,24 +54,51 @@ class LibIgnore {
     char *name;
     char *real_name;  // target of symlink
     bool loaded;
+    uptr ignored_code_range_id;
   };
 
   struct LibCodeRange {
     uptr begin;
     uptr end;
   };
 
+  // Marks a range as loaded by utilizing the least significant bit of the code
+  // range. Assumes the start of the code range is 2-byte aligned.
+  struct LibLoadedCodeRange {
+    uptr begin() const { return begin_ << 1; }
+    void begin(uptr begin) {
+      CHECK_EQ(begin & 0x1, 0);
+      begin_ = begin >> 1;
+    }
+
+   private:
+    uptr begin_ : 63;
+
+   public:
+    uptr loaded : 1;
```

Comment:
I am no sure understand why we need to track loaded.
Can we just set `begin = end = 0` ?

Reply:
Sounds good, I thought we might even get away with just setting `end = 0`, WDYT?